packages/react-components/react-toast/src/components/Toaster.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * ⚠️ This is temporary and WILL be removed
+ */
+
+import * as React from 'react';
+import { Portal } from '@fluentui/react-portal';
+import { useToaster, getPositionStyles } from '../state';
+import { Toast } from './Toast';
+import { makeStyles, mergeClasses } from '@griffel/react';
+
+const useStyles = makeStyles({
+  container: {
+    position: 'fixed',
+  },
+});
+
+export const Toaster: React.FC = () => {
+  const { getToastsToRender, isToastVisible } = useToaster();
+
+  const styles = useStyles();
+
+  return (
+    <Portal>
+      <div>
+        {getToastsToRender((position, toasts) => {
+          return (
+            <div key={position} style={getPositionStyles(position)} className={mergeClasses(styles.container)}>
+              {toasts.map(({ content, ...toastProps }) => {
+                return (
+                  <Toast
+                    {...toastProps}
+                    key={`toast-${toastProps.toastId}`}
```

Comment:
```suggestion
                    key={toastProps.toastId}
```

`toastId` is a string, you don't need a prefix there